svc/butler-srv/srv/acquire.go

```diff
@@ -0,0 +1,40 @@
+package srv
+
+import (
+	"encoding/json"
+	"github.com/Azeite-da-Quinta/jigajoga/libs/go/token"
+	"net/http"
+	"time"
+)
+
+func AcquireHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "POST" {
+		decoder := json.NewDecoder(r.Body)
+		var user token.Data
+		defer r.Body.Close()
+		err := decoder.Decode(&user)
+		if err != nil {
```

Comment:
Try to avoid indentation with guards (if blocks) that return immediately. 

Something like: 

```go
		if err != nil {
			slog.Info("ws: failed to XXXX", slog.String("err", err.Error()))

			http.Error(w,
				http.StatusText(http.StatusBadRequest),
				http.StatusBadRequest,
			)
			return //<- that way, the following code will not be indented in an else block.
		}
```